android-ktx/lib/build.gradle

```diff
@@ -185,10 +185,10 @@ dependencies {
     // androidx.work:work-runtime-ktx:2.8.0 requires 1.3.0
     compileOnly 'androidx.annotation:annotation:1.3.0'
 
-    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4"
+    compileOnly "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4"
```

Comment:
Jeff: How does this work?  Who provides the library at runtime?

Reply:
This is just to filter the coroutines dependency from being published as an explicit runtime dependency in both the .pom and .module, which is what you were doing already when manually filtering the published pom dependencies.

By specifying the dependency as `compileOnly`, the library consumer needs to provide the dependency if they make use of any of the CBL KTX APIs that depend on **coroutines-core** APIs (same with **work-runtime-ktx**). But it makes the dependency optional if they don't use those APIs.

The tests add the dependencies as `androidTestImplementation`, so they're added to the test binaries.

Again, this is how the behavior was already, this just makes it explicit and allows it to be the same in both the .pom and the .module metadata.